### ISSUE TITLE
[csstools] Create follow_up.md

### ISSUE THREAD

**Issue by karlalopez** [gdisf/teaching-materials#421]
1. Create `follow_up.md` in the lesson folder
2. Add content:

```
Thanks for attending the workshop, and thank you to our TAs [TODO: TA names] for being such great helpers. We’d love your feedback on the workshop in this survey. [TODO: Add link to survey]

The slides are available at:
http://www.teaching-materials.org/csstools/
We also discussed: [Chrome Canary](https://www.google.com/chrome/browser/canary.html) and [Firefox Beta](https://www.mozilla.org/en-US/firefox/channel/), [Chrome Device Emulation Mode](https://developer.chrome.com/devtools/docs/device-mode).

If any of you have existing website or blogs, think about how you can apply these tools to your blogs. Maybe check how they look in print mode or on your phone and see how you can improve them, and try using a framework like Bootstrap as the base CSS for your site.

For those of you who want to continue practicing CSS, here are some resources:

- [Treehouse: CSS](https://teamtreehouse.com/learn/css)
- [CodeSchool: HTML/CSS](https://www.codeschool.com/paths/html-css)
- [GDI CSS3 Slides](http://www.teaching-materials.org/css3/#slide1)

To keep up to date with things happening in CSS, I recommend subscribing to the [HTML5 weekly newsletter](http://html5weekly.com/) and [CSS tricks blog](https://css-tricks.com/). 

Stay in touch through [Facebook](https://www.facebook.com/gdisf/), [Twitter](https://twitter.com/gdisf), and [Slack](http://gdisf.slack.com). (Not a member? [Sign up for GDI SF Slack here](http://gdisf-slack.herokuapp.com)). Hope to see you at future workshops!
```

3. On /index.html, look for the **CSS Tools & Techniques** row and add the link to `follow_up.md`:
```
| <a href="https://github.com/gdisf/teaching-materials/blob/csstools/follow_up.md">Follow Up Email</a>
```

4. Test the links

**Comment by tarzioo** [comment]
Working on this

**Comment by verythorough** [comment]
Closed by #536. Thanks!
